docs.json

```diff
@@ -45,7 +45,7 @@
   "tabs": [
     {
       "id": "docs",
-      "title": "Docs",
+      "title": "Open Source",
```

Comment:
Name is a bit ambiguous for me, and "Open Source Package" is rather lengthy.
We can cross-reference the cloud in all pages, if it's an exposure issue.